### PR TITLE
Feature/62727549/swagger documentation

### PR DESCRIPTION
**Pull request by habibaudu**
#### What does this PR do?

-  Document API With Swagger

#### Description of Task to be completed?

- [x] Create a JSON Object of our Api
- [x] Convert JSON Object to swagger 
- [x] installs swagger-ui-exptress to serve the docs

#### How should this be manually tested?

Url:  `/api-docs/#/`

#### What are the relevant pivotal tracker stories?

[#162727549](https://www.pivotaltracker.com/story/show/162727549)

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,8 +1,11 @@
 
 import express from 'express';
+import swaggerUi from 'swagger-ui-express';
 import session from 'express-session';
 import cors from 'cors';
 import dotenv from 'dotenv';
+import swaggerDocument from './Learn Ground Api Documentation-Swagger20.json';
```
Comment (oreoluwade) [inline app.js]: Can you rename this json file? 

Could be lowercased all through with the spaces replaced with hyphens. Unless there are strict restrictions regarding the naming.

Comment (habibaudu) [reply app.js]: ok. rightway

**Comment by oreoluwade** [comment]
Regarding signup, in the `json` document, there are `isAdmin` and `isMentor` as available signup parameters too. Is there a reason those ones were not included? Or perhaps because they are not absolutely required?

Ultimately, I'd check the others endpoints for anything that may have been missed

**Comment by habibaudu** [comment]
> Regarding signup, in the `json` document, there are `isAdmin` and `isMentor` as available signup parameters too. Is there a reason those ones were not included? Or perhaps because they are not absolutely required?
> 
> Ultimately, I'd check the others endpoints for anything that may have been missed

They where not included because they are not required to signup.

**Comment by oreoluwade** [comment]
> > Regarding signup, in the `json` document, there are `isAdmin` and `isMentor` as available signup parameters too. Is there a reason those ones were not included? Or perhaps because they are not absolutely required?
> > Ultimately, I'd check the others endpoints for anything that may have been missed
> 
> They where not included because they are not required to signup.

Okay. Cool.

**Review by oreoluwade — APPROVED** [review]

**Pull request merged**